x-pack/plugins/apm/public/components/shared/Links/apm/service_transactions_overview_link.tsx

```diff
@@ -34,7 +38,7 @@ export function ServiceOrTransactionsOverviewLink({
 
   return (
     <APMLink
-      path={`/services/${serviceName}`}
+      path={getServicePath(serviceName)}
       query={pickKeys(urlParams as APMQueryParams, ...persistedFilters)}
       {...rest}
     />
```

Comment:
To cut down on the number of abstractions I suggest we stop using `APMLink` within helper link functions (all of the helpers in `/shared/Links/apm/`).

Instead we should be able to to this:

```ts
  const href = useServiceOrTransactionsOverviewHref(serviceName)
  return <EuiLink href={href} {...rest} />;
```
This also means we won't need `getServicePath`.

Do you mind updating the other helper functions accordingly?